GetandTake/Program.cs

```diff
@@ -5,7 +5,15 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-builder.Services.AddRazorPages();
+builder.Services
+    .AddRazorPages()
+    .AddRazorRuntimeCompilation()
+    .AddMvcOptions(options =>
+{
```

Comment:
Let's add tabulation here to align delegate with it's method call, like:
```suggestion
     .AddMvcOptions(options =>
     {
           ....
     });
```